src/test/java/org/jenkinsci/plugins/workflow/properties/PipelineTriggersJobPropertyTest.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016, CloudBees, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ *
+ */
+package org.jenkinsci.plugins.workflow.properties;
+
+import hudson.diagnosis.OldDataMonitor;
+import hudson.model.Item;
+import hudson.model.Items;
+import hudson.model.Saveable;
+import hudson.triggers.TimerTrigger;
+import hudson.triggers.Trigger;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty;
+import org.junit.After;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.BuildWatcher;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.recipes.LocalData;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class PipelineTriggersJobPropertyTest {
+    @ClassRule
+    public static BuildWatcher buildWatcher = new BuildWatcher();
+    @Rule
+    public JenkinsRule r = new JenkinsRule();
+
+    /**
+     * Needed to ensure that we get a fresh {@code startsAndStops} with each test run. Has to be *after* rather than
+     * *before* to avoid weird ordering issues with {@code @LocalData}.
+     */
+    @After
+    public void resetStartsAndStops() {
+        MockTrigger.startsAndStops = new ArrayList<>();
+    }
+
+    @Test
+    public void loadCallsStartFalse() throws Exception {
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
+        MockTrigger t = new MockTrigger();
+        p.addTrigger(t);
+        p.save();
+        p = (WorkflowJob) Items.load(p.getParent(), p.getRootDir());
```

Comment:
:ant: More realistic simulations would include:

* use `@LocalData` with a `config.xml` representing an existing trigger
* add as above, but use `RestartableJenkinsRule` and do this test after the restart
* use `AbstractItem.doReload`